### DISCUSSION TITLE
Which PhpSpreadsheet Version is Compatible with PHP 8.0 in XAMPP?

### DISCUSSION THREAD

**Discussion by kekincai** [PHPOffice/PhpSpreadsheet#4219] [Q&A]
Hello everyone,

I’m currently running PHP 8.0.30 on XAMPP, and I’m unable to use Composer to manage packages. I want to use PhpSpreadsheet to read and manipulate Excel files, but I need to know which version of PhpSpreadsheet is compatible with PHP 8.0.

Could anyone recommend the correct version of PhpSpreadsheet that works well with PHP 8.0? Also, since I can’t use Composer, any advice on manually installing it would be greatly appreciated.

Thank you in advance!

**Comment by oleibman** [comment]
Why can't you use Composer? See https://github.com/PHPOffice/PhpSpreadsheet/issues/3221#issuecomment-1469438680

And, not that it necessarily matters, why are you still using Php 8.0, which stopped receiving even security updates over a year ago?

**Comment by oleibman** [comment]
Closing. No response in 2.5 months. No reason to keep open.
